.gitignore

```diff
@@ -1,3 +1,24 @@
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
+
+# dependencies
+/node_modules
+/.pnp
+.pnp.js
+
+# testing
+/coverage
+
+# production
+/build
+
+# misc
+.DS_Store
```

Comment:
These are all below

Reply:
The `.DS_Store` was but not the `/build` - only a specific sub folder of it.